pkg/k8s/util.go

```diff
@@ -23,29 +49,111 @@ func PodIsStatic(pod *v1.Pod) bool {
 	return ok && configSource == "file"
 }
 
-// CalculatePodsRequestsTotal returns the total capacity of all pods
-func CalculatePodsRequestsTotal(pods []*v1.Pod) (resource.Quantity, resource.Quantity, error) {
-	memoryRequests := *k8s_resource.NewMemoryQuantity(0)
-	cpuRequests := *k8s_resource.NewCPUQuantity(0)
+// CalculatePodsRequestedUsage returns the total capacity of all pods
```

Comment:
Can we update this comment to be more accurate - i.e. returns total requested resources and largest request?